sdk/src/main/java/com/qonversion/android/sdk/internal/mappers/Mapper.kt

```diff
@@ -2,8 +2,11 @@ package com.qonversion.android.sdk.internal.mappers
 
 interface Mapper<T> {
     @Throws(IllegalStateException::class)
-    fun toMap(value: T): Map<String, Any?>
+    fun toMap(value: T): Map<String, Any?> { return emptyMap() }
 
     @Throws(IllegalStateException::class)
-    fun fromMap(data: Map<String, Any?>): T
+    fun fromMap(data: Map<String, Any?>): T?
+
+    @Throws(IllegalStateException::class)
+    fun arrayFromMap(data: Map<*, *>): List<T> { return emptyList() }
```

Comment:
what is the use case of this method?

Reply:
```
{
    data: "some string",
    data2: [
            {}, {}, {}
           ]
}
```

Reply:
And what is the expected result?

Reply:
An array of objects (T)
Entitlements, for example